extensions/kotlin/runtime/pom.xml

```diff
@@ -19,6 +19,20 @@
                 <groupId>io.quarkus</groupId>
                 <artifactId>quarkus-bootstrap-maven-plugin</artifactId>
                 <configuration>
+                    <parentFirstArtifacts>
```

Comment:
I had started to realize late yesterday the answer probably lay in loading parent first.  I was going to hit you up for input today actually.  But i had no idea this mechanism existed.    Pretty brilliant.